neo.UnitTests/protocol.json

```diff
@@ -1,5 +1,5 @@
 {
   "ProtocolConfiguration": {
-    "SecondsPerBlock": 2
+    "MillisecondsPerBlock": 2000
```

Comment:
We don't need to change this, is a different value, one thing is store the timestamp in ms, and other be able to set 1788 ms per block (for example) always we will use seconds

Reply:
What do you suggest, @shargon?

I prefer leave configuration in milliseconds.
Otherwise we can set a flag if user is setting in seconds or milli.

Reply:
Do you think that the time per block need to be in ms?

Reply:
Yes, we need everything to be logged in `ms`, both consensus and blocks timestamps.

The idea is not having 1ms, but, at least, privatenets running with 100~500ms.